### PR TITLE
Added functions to take selection and get API results into searchResults array

### PR DESCRIPTION
**Pull request by kevin-ivy**

### CODE REVIEW

File: assets/js/keys.js [hunk 1]
```diff
@@ -1,56 +1,128 @@
...
+function getAmazonUrl() {
+	fetch(`https://axesso-axesso-amazon-data-service-v1.p.rapidapi.com/amz/amazon-search-by-keyword-asin?sortBy=relevanceblender&domainCode=com&keyword=${searchTerm}&page=1`, {
+		"method": "GET",
+		"headers": {
+			"x-rapidapi-host": amazonHost,
+			"x-rapidapi-key": axsessoKey
+		}
+	})
+	.then(response => {
+		if (response.ok) {
```
Comment (melissarpowell) [inline assets/js/keys.js:38]: We should probably add an else statement for each request to handle not ok responses?

File: assets/js/keys.js [hunk 2]
```diff
@@ -1,56 +1,128 @@
...
+		prime: product.prime,
+		url: `https://www.amazon.com/dp/${product.asin}`
+	};
+}
+
+function makeWalmartProduct(product) {
+	return {
+		title: product.productTitle,
+		price: product.price || '',
+		availability: product.available,
```
Comment (melissarpowell) [inline assets/js/keys.js:140]: Do we need to call `isAvailable()` here too?

File: index.html [hunk 3]
```diff
@@ -27,58 +31,93 @@
...
+        <div class="row" style= "margin-top:10%;">
+          
+          <h1 class="title">Paper Locator <img class="logo two columns" src="./assets/images/logo.svg" alt="TP logo"></h1>
+
+       </div>
+       <div style="margin-left: 25%;">
+       <h4>It has become dificult finding essential paper products.<br/>
+       With Paper Locator you can</h4>
+       <ul style="margin-left:20%;">
+         <li>Pick your quarentine essentials. We will even help you find hand sanitizer!</li>
```
Comment (melissarpowell) [inline index.html]: typo - "quarantine"

File: index.html [hunk 4]
```diff
@@ -27,58 +31,93 @@
...
+    <!--Instructional greeting modal-->
+    <div class="modal" id="modal" style="margin-left=15%;"> 
+      <div class="container modal-content">
+        <div class="row" style= "margin-top:10%;">
+          
+          <h1 class="title">Paper Locator <img class="logo two columns" src="./assets/images/logo.svg" alt="TP logo"></h1>
+
+       </div>
+       <div style="margin-left: 25%;">
+       <h4>It has become dificult finding essential paper products.<br/>
```
Comment (melissarpowell) [inline index.html]: Typo - "difficult"

File: index.html [hunk 5]
```diff
@@ -27,58 +31,93 @@
...
+       With Paper Locator you can</h4>
+       <ul style="margin-left:20%;">
+         <li>Pick your quarentine essentials. We will even help you find hand sanitizer!</li>
+         <li>Instantly view the availabilty and pricings from top online retailers</li>
```
Comment (melissarpowell) [inline index.html]: Typo - "availability"

File: index.html [hunk 6]
```diff
@@ -27,58 +31,93 @@
...
       </tr>
     </thead>
     <tbody>
       <tr>
-        <td>Amazon</td>
-        <td>$13.68</td>
-        <td>12</td>
+       
+        <td>Amazon  <img class="prime-icon" src="./assets/images/prime-icon.svg" alt="prime"></td>
+        <td>Charmon 12 roll</td>
```
Comment (melissarpowell) [inline index.html]: Typo - "Charmin"

**Review by melissarpowell — CHANGES_REQUESTED** [review]
Looks great, just a few typos.

File: assets/js/keys.js (hunk 1 above)
Comment (bradleypaul) [reply assets/js/keys.js:38]: Use the .catch function to get any errors (400 and 500 class are most common)

File: assets/js/keys.js (hunk 2 above)
Comment (bradleypaul) [reply assets/js/keys.js:140]: Fortunately, no. Walmart has a boolean property 'available' on the product response so we can just save it to the product object.

**Review by melissarpowell — APPROVED** [review]

**Pull request merged**